packages/plugin-css/test/js.test.ts

```diff
@@ -11,61 +11,62 @@ import css from '../src/index.js';
 
 describe('Node.js API', () => {
   describe('token types', () => {
-    it.each([
-      /*'boolean',*/ 'border' /*'color', 'dimension', 'gradient', 'shadow', 'string', 'typography', 'transition'*/,
-    ])('%s', async (dir) => {
-      const output = 'actual.css';
-      const cwd = new URL(`./fixtures/type-${dir}/`, import.meta.url);
-      const config = defineConfig(
-        {
-          plugins: [
-            css({
-              filename: output,
-              variableName: (name) => makeCSSVar(name, { prefix: 'ds' }),
-              modeSelectors: [
-                {
-                  mode: 'light',
-                  tokens: ['color.*', 'gradient.*'],
-                  selectors: ['@media (prefers-color-scheme: light)', '[data-color-theme="light"]'],
-                },
-                {
-                  mode: 'dark',
-                  tokens: ['color.*', 'gradient.*'],
-                  selectors: ['@media (prefers-color-scheme: dark)', '[data-color-theme="dark"]'],
-                },
-                {
-                  mode: 'light-colorblind',
-                  tokens: ['color.*'],
-                  selectors: ['[data-color-theme="light-colorblind"]'],
-                },
-                {
-                  mode: 'light-high-contrast',
-                  tokens: ['color.*'],
-                  selectors: ['[data-color-theme="light-high-contrast"]'],
-                },
-                { mode: 'dark-dimmed', tokens: ['color.*'], selectors: ['[data-color-theme="dark-dimmed"]'] },
-                {
-                  mode: 'dark-high-contrast',
-                  tokens: ['color.*'],
-                  selectors: ['[data-color-theme="dark-high-contrast"]'],
-                },
-                { mode: 'dark-colorblind', tokens: ['color.*'], selectors: ['[data-color-theme="dark-colorblind"]'] },
-                { mode: 'desktop', selectors: ['@media (width >= 600px)'] },
-              ],
-            }),
-          ],
-        },
-        { cwd },
-      );
-      const tokensJSON = new URL('./tokens.json', cwd);
-      const { tokens, sources } = await parse([{ filename: tokensJSON, src: fs.readFileSync(tokensJSON, 'utf8') }], {
-        config,
-      });
-      const result = await build(tokens, { sources, config });
-      expect(result.outputFiles.find((f) => f.filename === output)?.contents).toMatchFileSnapshot(
-        fileURLToPath(new URL('./want.css', cwd)),
-      );
-    });
+    it.each(['boolean', 'border', 'color', 'dimension', 'gradient', 'shadow', 'string', 'typography', 'transition'])(
```

Comment:
🤦 looks like I was debugging something and accidentally committed it. thanks for re-enabling